### DISCUSSION TITLE
GitHub pages showing old version of a css file

### DISCUSSION THREAD

**Discussion by Emorque** [community/community#146863] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

This may be a bug but after pushing a new commit, the styling of my website on my local machine is difference than the deployed page. Looking at source, I see that the css files is from an older commit. My repo has the lastest css file, but pages seems to have an older version, so styling is not updated. 

Deployed projects.css file:
![image](https://github.com/user-attachments/assets/8ce909c0-0abc-4106-a143-cc98a01a79ea)

projects.css file on my repo (the lastest one that I want pages to use)
![image](https://github.com/user-attachments/assets/d276eea6-b6d6-4a67-bfa4-bfec1b125cd7)

**Comment by MindfulLearner** [comment]
Could be youre not in the main branch?

**Reply by Emorque** [reply]
I figured it out, it was my browser's caching. Thanks for the tip though

**Comment by Emorque** [comment, marked as answer]
I realized that it was my own browser's caching that was the issue. Just deleting the last hour of browsing data fixed it, and now the site has the updated files.
